### PR TITLE
Add CLI command to validate Schema

### PR DESCRIPTION
**Pull request by MichaelDemeyer**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Documented?   | no
| Fixed tickets | 
| License       | MIT

Add CLI command to easily run $schema->assertValid()

### CODE REVIEW

File: src/Command/ValidateCommand.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        parent::__construct();
+        $this->typeResolver = $typeResolver;
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('graphql:validate')
+            ->setDescription('Validate schema')
+        ;
```
Comment (mcg-web) [inline src/Command/ValidateCommand.php]: The bundle allow multiple schema definition so this should also allow it, [see this command](https://github.com/overblog/GraphQLBundle/blob/master/src/Command/GraphQLDumpSchemaCommand.php#L40-L45) for a concrete example.

File: src/Command/ValidateCommand.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $query = $this->typeResolver->resolve('Query');
```
Comment (mcg-web) [inline src/Command/ValidateCommand.php]: You should not assume that all Query or Mutation are named so. [See how to get an existing scheme](https://github.com/overblog/GraphQLBundle/blob/master/src/Command/GraphQLDumpSchemaCommand.php#L108).

File: src/Command/ValidateCommand.php [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $schemaName = $input->getOption('schema');
+        $schema = $this->getRequestExecutor()->getSchema($schemaName);
+
+        try {
+            @$schema->assertValid();
```
Comment (mcg-web) [inline src/Command/ValidateCommand.php]: Can you comment why are you using `@` here please?

Comment (MichaelDemeyer) [reply src/Command/ValidateCommand.php]: I rework this one to use a cleaner and more explicit method :
Warning::suppress(true);

File: src/Command/ValidateCommand.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+final class ValidateCommand extends Command
+{
+    use RequestExecutorLazyLoaderTrait;
+
+    private $typeResolver;
+
+    public function __construct(TypeResolver $typeResolver)
+    {
+        parent::__construct();
+        $this->typeResolver = $typeResolver;
```
Comment (mcg-web) [inline src/Command/ValidateCommand.php]: It seems you don't used this anymore, I'm a right? Can you remove it and construct if this is the case.

Comment (MichaelDemeyer) [reply src/Command/ValidateCommand.php]: Right, I removed it.

**Pull request merged**